recipes/gstreamer/all/conandata.yml

```diff
@@ -1,4 +1,10 @@
 sources:
+  "1.22.3":
+    url: "https://gstreamer.freedesktop.org/src/gstreamer/gstreamer-1.22.3.tar.xz"
```

Comment:
I am fine with changing the URL to the gstreamer. instead of gitlab, because the later is non-navigable, but the former is.

Reply:
GitLab/GitHub auto-generated archives cannot be trusted.